src/services/auth/auth-router.ts

```diff
@@ -154,7 +154,7 @@ authRouter.get("/dev/", (req, res) => {
 
 authRouter.get(
     "/corporate",
-    RoleChecker([Role.Enum.ADMIN], true),
+    RoleChecker([Role.Enum.ADMIN]),
```

Comment:
I think we'll want to keep `weakVerification` on this one so that we can use it on the website's sponsors section